apps/mobile/src/utils/transactions.tsx

```diff
@@ -43,6 +45,14 @@ export const getTransactionType = ({ txInfo }: { txInfo: Transaction['txInfo'] }
     return ETxType.STAKE_DEPOSIT
   }
 
+  if (isStakingTxExitInfo(txInfo)) {
+    return ETxType.STAKE_WITHDRAW_REQUEST
+  }
+
+  if (isStakingTxWithdrawInfo(txInfo)) {
+    return ETxType.STAKE_EXIT
```

Comment:
nit: Naming is a bit inconsistent. I would suggest either `STAKE_WITHDRAW` or `STAKE_EXIT_REQUEST`